src/azure_devtools/scenario_tests/const.py

```diff
@@ -12,3 +12,5 @@
 ENV_LIVE_TEST = 'AZURE_TEST_RUN_LIVE'
 ENV_SKIP_ASSERT = 'AZURE_TEST_SKIP_ASSERT'
 ENV_TEST_DIAGNOSE = 'AZURE_TEST_DIAGNOSE'
+# Keep crime scene (resources) when a test case fails. It is super useful for debugging.
+ENV_RECOVER = 'AZURE_TEST_RECOVER'
```

Comment:
how about make it straightforward `AZURE_TEST_FAIL_NOCLEANUP`?

Reply:
Updated